README.md

```diff
@@ -37,7 +37,8 @@ ActivityView.show({
   text: "Text you want to share",
   url: "URL you want to share",
   imageUrl: "Url of the image you want to share/action",
-  image: "Name of the image in the app bundle"
+  image: "Name of the image in the app bundle",
+  anchor: React.findNodeHandle(this.refs.share), // where to show popup on ipad?
```

Comment:
Minor, but can you update the comment to:

`Where you want the share popup to point to on iPad`